pages/common/csvquote.md

```diff
@@ -0,0 +1,20 @@
+# csvquote
+
+> Enables common utilities such as `cut` and `awk` to work correctly with CSV files containing quoted delimiters and newlines.
+> More information: <https://github.com/dbro/csvquote>.
+
+- Replace quoted special characters in normal csv file:
```

Comment:
CSV should be in capital here. And what is a normal CSV file ? And it should also be "in _a_ ... file"

Reply:
@agnivade I suggested changes to speed the process up. Thanks for writing this down!